src/mudlet-lua/lua/geyser/GeyserContainer.lua

```diff
@@ -190,6 +190,41 @@ function Geyser.Container:lower ()
 	lowerWindow(self.name)
 end
 
+function Geyser.Container:raiseAll(container, me)
+  me = me or true
+  container = container or self
+  -- raise myself
+  if me then 
+    container:raise() 
+  end
+  
+  local v
+  for i=1,#container.windows do
+    v = container.windows[i]
+    container.windowList[v]:raise()
+    container.windowList[v]:raiseAll(container.windowList[v], false)
+  end
+end
+
+local function createReverseTable(container)
+ local v
+  Geyser.Container.reverseTable = Geyser.Container.reverseTable or {}
+    for i=1,#container.windows do
+    v = container.windows[i]
```

Comment:
Mind fixing up the indenting - it goes into 3 levels despite logically being on one.